pkg/server/routes.go

```diff
@@ -68,12 +68,12 @@ func (s *Server) routes(
 	}
 
 	// set up CEDAR client
-	cedarEasiClient := local.NewCedarEasiClient(s.logger)
+	var cedarEasiClient cedareasi.Client = local.NewCedarEasiClient(s.logger)
 	if !(s.environment.Local() || s.environment.Test()) {
 		// check we have all of the configs for CEDAR clients
 		s.NewCEDARClientCheck()
 
-		cedarEasiClient := cedareasi.NewTranslatedClient(
```

Comment:
This colon was the crux of the bug... Because it was assigning to a NEW variable called `cedarEasiClient` within the scope of the `if` statement, rather than replacing the value on that variable in the outer scope. D'oh!